### ISSUE TITLE
The secret key should go into a separate file

### ISSUE THREAD

**Issue by chris-martin** [cachix/cachix#74]
The secret key currently goes into the same file as the rest of the configuration; it would be nice to keep it separate so that the rest of the config can be shared publicly.

**Comment by domenkozar** [comment]
Why is sharing Cachix config file publicly useful? I'm just wondering :)

In any case since it's Dhall you could modify the config file to import secrets yourself and Cachix should still work. Not tested.

**Comment by chris-martin** [comment]
I've realized since I made this comment that I don't actually know what goes in the Cachix file other than secrets, so maybe this isn't actually an issue. For me, making stuff public is just a matter of convenience, because it means I can keep it in the version-controlled part of [my home directory](https://github.com/chris-martin/home).

Does Cachix programmatically edit the config file? I'm not used to assuming that that would work after I've edited it.
